DIPs/dip10ss.md

```diff
@@ -0,0 +1,284 @@
+# Enum and Function Parameter Attributes
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0 (edited by DIP Manager)                                       |
+| Author:         | [@skl131313](https://github.com/skl131313)                      |
+| Implementation: | [(Partial) Function UDAS](https://github.com/dlang/dmd/pull/7576) |
```

Comment:
I think you'll want to add link to the enum PR here: https://github.com/dlang/dmd/pull/6161